### ISSUE TITLE
Template values for tenant property in Loki output

### ISSUE THREAD

**Issue by ecusnir** [banzaicloud/logging-operator#648]
> Bugs should be filed for issues encountered whilst operating logging-operator.
> You should first attempt to resolve your issues through the community support
> channels, e.g. Slack, in order to rule out individual configuration errors. [#logging-operator](https://slack.banzaicloud.io/)
> Please provide as much detail as possible.

**Describe the bug**:
When using Loki as output, the tenant property is render in the fluentd config surrounded by quotes, thus making it impossible to use template values:
`tenant "${$.kubernetes.labels.tenant}"`
**Expected behaviour**:
As stated in [Loki's plugin docs](https://grafana.com/docs/loki/latest/clients/fluentd/#tenant), the tenant shouldn't be enclosed in quotes when setting a template value.
**Steps to reproduce the bug**:
Generate the Output plugin using the following spec.
```
spec:
  loki:
    tenant: ${$.kubernetes.labels.tenant}
```
**Environment details**:
- Kubernetes version (e.g. v1.15.2): v1.16.3
- Cloud-provider/provisioner (e.g. AKS, GKE, EKS, PKE etc): on-prem
- logging-operator version (e.g. 2.1.1): 3.8.0
- Install method (e.g. helm or static manifests): helm

/kind bug

**Comment by itsLucario** [comment]
This issue still exists and I'm unable to find a workaround. 

Have you found any work around for this @ecusnir ?

**Comment by ecusnir** [comment]
Not really. I can’t properly use the tenant label right now and it’s really inconvenient for my project.

**Comment by envybee** [comment]
I've found the fix for this (worked for me at least)... You'd need to set the placeholder as a "tag" in the "buffer" section as well:
```
tenant: "${$.kubernetes.labels.tenant}"
buffer:
  tags: $.kubernetes.labels.tenant
```

**Comment by itsLucario** [comment]
That's a great finding. I'll definitely try it out. Thanks @envybee

**Comment by ahma** [comment]
I will close this issue for now, if the solution above not working fully we can reopen it.
